trainingMaquetado/maquetado.css

```diff
@@ -6,6 +6,30 @@
   margin-bottom: 20px;
 }
 
+.subscribe {
+  background-color: Transparent;
+  border-color: white;
+  border-radius: 10px;
+  border-style: solid;
+  border-width: 0.5px;
+  color: white;
+  font-size: 16px;
+  font-weight: bold;
+  height: 40px;
+  margin-bottom: 20px;
+  width: 120px;  
+}
+
+.insert {
+  border-color: white;
+  border-radius: 10px;
+  border-style: solid;
+  font-size: 16px;
+  height: 30px;
+  text-align: center;
+  width: 280px;
```

Comment:
this might break small layouts, to avoid this kind of problems use:
```
width: 100%;
max-width: 280px;
```

this will make the input shrink if the width is less than 280px